<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-K6TC2A2S" name="Test" tests="9" failures="1" timestamp="22 Nov 2023 11:54:49 GMT" time="0.048" errors="0">
  <testcase name="Packinglist" time="0.002" classname="testNG.Annotations"/>
  <testcase name="userAccounts" time="0.0" classname="testNG.Annotations"/>
  <testcase name="Demo" time="0.001" classname="testNG.Pgm1"/>
  <testcase name="Second" time="0.004" classname="testNG.Pgm1">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at testNG.Pgm1.Second(Pgm1.java:20)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Second -->
  <testcase name="userinventory" time="0.0" classname="testNG.Annotations"/>
  <testcase name="Basic" time="0.007" classname="testNG.Dataprovider"/>
  <testcase name="dataprovide" time="0.0" classname="testNG.Dataprovider"/>
  <testcase name="dataprovide" time="0.0" classname="testNG.Dataprovider"/>
  <testcase name="dataprovide" time="0.0" classname="testNG.Dataprovider"/>
</testsuite> <!-- Test -->
